lib/instances/TestService_spec.lua

```diff
@@ -1,10 +1,29 @@
 local Instance = import("../Instance")
 
+--TODO: Replace Instance.new("TestService") with game:GetService("TestService")
```

Comment:
Using `game:GetService` is partly problematic because then we need to construct an entire "game" just to test a piece of the game. If a service depends on other services, that might be useful.

We can leave this TODO out for now I think, unless you have a strong argument otherwise.

Reply:
Sounds fine to me, but maybe in the future if we decide we want to be more realistic to Roblox we can make "Instance.newForced" or something to ignore all errors about not being able to create a service.

Reply:
That sounds like a solid plan, you should open an issue about that before we forget that idea!

Reply:
Done, #13